src/posts/2021-09-27-Using-focus-style-for-accessibility

```diff
@@ -0,0 +1,133 @@
+---
+title: Using focus style for accessibility
+description: "A simple guide on how to make your website more accessible with focus style."
+category: How-to
+author: Ezekiel Lawson
+date: 2021-09-27 
+further_reading:
+  - title: Style focus
+    url: https://https://web.dev/style-focus/
+    source:  Web Dev
+    year: 2018
+  - title: Designing usable focus indicators
+    url: https://www.deque.com/blog/give-site-focus-tips-designing-usable-focus-indicators/
+    source:  Caitlin Geier
+    year: 2016
+  - title: Never remove CSS outlines
+    url: https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/
+    source: Guilherme Simões
+    year: 2013
+tags:
+  - How-to
+---
+
+While accessing any digital interface, a focus indicator is important it highlights the active intractable component in a webpage. 
+
+## What is a focus style?
+
+A focus is a pseudo-class used to indicate or highlights an intractable component in a webpage.
+
+## Why is it important?
+
+A focus style is necessary because some people use a mouse or a touch device to access a website some rely on using a keyboard to navigate on a website. Having a focus indicator will improve the usability of your website and makes it accessible, It also tells us the interactive element we are focused on.
+
+## How to apply focus style
+
+Preceding code is an example of how we can apply focus style to our elements:
+
+### Applying focus style to a link
+
+```CSS
+a:focus {
+  outline: 5px solid red;
+  }
+```
+
+The following code example is a link that has a custom style for it focus state:
+
+### Code explanation
+
+In the following code example, we added focus to our link and gave it an outline of solid red with a value of 5px. Please note the focused style is applied only to the link tag itself.
```

Comment:
I am confused about this and the previous paragraph. I think they need to be combined.